office/libreoffice/Portfile

```diff
@@ -0,0 +1,315 @@
+                    sha256  f94fb0ad8216f97127bedef163a45886b43c62deac5e5b0f5e628e234220c8db \
+                    size    707920 \
+                    libzmf-0.0.2.tar.xz \
+                    rmd160  b0be3dbe0f6bf4565c1668a236850601616f625f \
+                    sha256  27051a30cb057fdb5d5de65a1f165c7153dc76e27fe62251cbb86639eb2caf22 \
+                    size    320952 \
+                    mdds-1.6.0.tar.bz2 \
+                    rmd160  003175ca1b3d1293a6f144490de94eba72cb1697 \
+                    sha256  f1585c9cbd12f83a6d43d395ac1ab6a9d9d5d77f062c7b5f704e24ed72dae07d \
+                    size    350406 \
+                    xmlsec1-1.2.30.tar.gz \
+                    rmd160  5de6607d95b852da826f99ca6ca89a81f71aaaf3 \
+                    sha256  2d84360b03042178def1d9ff538acacaed2b3a27411db7b2874f1612ed71abc8 \
+                    size    1989670 \
+                    884ed41809687c3e168fc7c19b16585149ff058eca79acbf3ee784f6630704cc-opens___.ttf \
+                    rmd160  00b1e296adf2caad44c49ad9743e5da7c771e480 \
+                    sha256  884ed41809687c3e168fc7c19b16585149ff058eca79acbf3ee784f6630704cc \
+                    size    207544
+
+depends_build-append \
+    port:autoconf \
+    port:automake \
+    port:bison \
+    port:cppunit \
+    port:flex \
+    port:gettext \
+    port:gperf \
+    port:intltool \
+    port:pkgconfig \
+    port:unixODBC
+depends_lib-append  \
+    port:apr \
+    port:apr-util \
+    port:boost \
+    port:bzip2 \
+    port:curl \
+    port:expat \
+    port:gdbm \
+    port:gettext \
+    port:graphite2 \
+    port:harfbuzz \
+    port:harfbuzz-icu \
+    port:hunspell \
+    port:hyphen \
+    port:icu \
+    port:jpeg \
+    port:lcms2 \
+    port:libcdr-0.1 \
+    port:libe-book \
+    port:libepoxy \
+    port:libetonyek \
+    port:libfreehand \
+    port:libiconv \
+    port:liblangtag \
+    port:libmspub \
+    port:libodfgen \
+    port:libpagemaker \
+    port:libpng \
+    port:librevenge \
+    port:libvisio-0.1 \
+    port:libwpd-0.10 \
+    port:libwpg-0.3 \
+    port:libwps \
+    port:libxml2 \
+    port:libxslt \
+    port:ncurses \
+    port:nspr \
+    port:nss \
+    path:lib/libssl.dylib:openssl \
+    port:ossp-uuid \
+    port:raptor2 \
+    port:readline \
+    port:redland \
+    port:serf1 \
+    port:sqlite3 \
+    port:xz \
+    port:zlib
+# Try to keep this in sync with the Python portgroup.
+set python_version 38
+depends_run-append \
+    port:python${python_version} \
+    port:py${python_version}-lxml
+
+if {${os.major} < 14} {
+    pre-fetch {
+        ui_error "${name} @${version} requires OS X 10.10 or later."
+        return -code error "incompatible OS X version"
+    }
+}
+
+patchfiles \
+    pdfread-cxx-undefined.patch \
+    system-python.patch \
+    unpack-sources-fix-for-bsd-find.patch
+
+set product_name LibreOffice
+
+pre-configure {
+    system "cd ${worksrcpath} && env NOCONFIGURE=1 ./autogen.sh"
```

Comment:
I tried to use `use_autoreconf yes` here instead but it failed due to undefined macro `AS_HELP_STRING`.